src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -82,39 +84,80 @@ export default class HexagonLayer extends Layer {
     super(props);
   }
 
+  getShaders() {
+    return this.props.fp64 ? {
+      vs: readFileSync(join(__dirname, './hexagon-layer-64-vertex.glsl'), 'utf8'),
+      fs: readFileSync(join(__dirname, './hexagon-layer-fragment.glsl'), 'utf8'),
```

Comment:
This construction will inline the fragment shader twice, adding a long string twice to the code. It could be better to do:
```
const vs = readFileSync(...
const vs64 = readFileSync(
const fs = readFileSync(
```
And then just return compact one liner objects built from those variables. It might be a little easier to read too?